mars/deploy/oscar/tests/modules/replace_op.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from mars.tensor.arithmetic.add import TensorAdd
+
+
+def _replace_op(ctx, op):
+    # change the op from TensorAdd to TensorSubtract.
+    type(op)._func_name = 'subtract'
+    executor = type(op).execute
+    return executor(ctx, op)
+
+
+print("replace TensorAdd to TensorSubtract for testing.")
```

Comment:
Is this print intended or just for test?

Reply:
This is a test print in a test module.